### DISCUSSION TITLE
Could .trivyignore file ignore files or directories?

### DISCUSSION THREAD

**Discussion by yansifw** [aquasecurity/trivy#4584] [Ideas]
### Description

It would be great to allow .trivyignore to ignore specific files or directories per project instead of using the --skip-dirs / --skip-files flags. When running trivy from the CI, each project could ignore some specific paths using .trivyignore instead of having to modify the OS command with the available flags.

What do you think?

Thanks!

### Target

Filesystem

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
You can use trivy.yaml
https://aquasecurity.github.io/trivy/v0.42/docs/references/configuration/config-file/
